.github/workflows/nightly.yaml

```diff
@@ -9,7 +9,7 @@ on:
 
 env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  GO_VERSION: '1.17'
+  GO_VERSION: '1.18'
```

Comment:
too soon to 1.19?

Reply:
yes 😄

Reply:
I mean.. no, but lets first have everything for once at the same version